packages/@webex/plugin-meetings/src/meeting/index.ts

```diff
@@ -5216,7 +5230,7 @@ export default class Meeting extends StatelessWebexPlugin {
       // @ts-ignore
       this.webex.internal.newMetrics.submitClientEvent({
         name: 'client.media.tx.start',
-        payload: {mediaType: data.type},
+        payload: {mediaType: data.type, shareInstanceId: this.shareInstanceId},
```

Comment:
hmm not sure about this one and the next 3 changes inside setupStatsAnalyzerEventHandlers()... we will now be sending `shareInstanceId` with all `client.media.rx/tx.start/stop` events, even the ones that have mediaType "audio" or "video" so nothing to do with sharing. I guess call analyzer would ignore it so probably it's not going to cause any harm, but can be a bit confusing and it's not very tidy. We really want to send the shareInstanceId only when mediaType is "share"